dev_scripts/test.py

```diff
@@ -0,0 +1,21 @@
+from redis import Redis
```

Comment:
this should be part of the test suite, not hidden away in `dev_scripts`

Reply:
I will make a fab script

Reply:
removed